COPY/usr/lib/systemd/system/evmserverd.service

```diff
@@ -6,6 +6,7 @@ After=syslog.target
 ExecStart=/bin/sh -c "/bin/evmserver.sh start"
 ExecStop=/bin/sh -c "/bin/evmserver.sh stop"
 Type=forking
+Restart=on-failure
```

Comment:
Do want to restart only on the specific failure codes?  It might be weird if the evmserverd exits on fatal signal (128+)